src/validation.cpp

```diff
@@ -902,8 +902,24 @@ bool MemPoolAccept::PreChecks(ATMPArgs& args, Workspace& ws)
     }
 
     // Check for non-standard witnesses.
```

Comment:
Small nit. This comment could be updated now that we're checking for both `nonstandard` and `stripped` witnesses

```suggestion
    // Check for witness-related policy violations.
```